test/api/CMakeLists.txt

```diff
@@ -11,7 +12,6 @@ endfunction()
 
 add_api_test(array_base test_array_base.cpp ${TEST_ONNX_DIR})
 add_api_test(assign test_assign.cpp ${TEST_ONNX_DIR})
-add_api_test(custom_op test_custom_op.cpp ${TEST_ONNX_DIR})
```

Comment:
should we still make a test for custom op that doesn't rely on the GPU?

Reply:
Added one `ref` target custom op test.